LICENSE.txt

```diff
@@ -1,21 +1,97 @@
-The MIT License (MIT)
-
-Copyright (c) 2019 Ryan Jackson
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+Business Source License 1.1
+
+Parameters
+
+Licensor:               Modern Treasury, Inc.
+Licensed Work:          LedgerSync
+                        The Licensed Work is (c) 2020 Modern Treasury, Inc.
+Additional Use Grant:   You may make use of the Licensed Work, provided that you
+                        may not use the Licensed Work for a Connector Service.
+
+                        A “Connector Service” is a commercial offering that
+                        allows third parties (other than your employees and
+                        contractors) to access the functionality of the Licensed
+                        Work by connecting to and exchanging information with
+                        general ledgers, accounting software, or other adaptors
+                        defined in the library that are controlled by such
+                        third parties.
+
+Notice
+
+The Business Source License (this document, or the “License”) is not an Open
+Source license. However, the Licensed Work will eventually be made available
+under an Open Source License, as stated in this License.
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+“Business Source License” is a trademark of MariaDB Corporation Ab.
```

Comment:
@ryanwjackson There are a few references to MariaDB in here, do we want to change them to ModernTreasury?

Reply:
@SeanBolt this License and TM is the copyright of MariaDB.  We must include this part in order to use it.